### PR TITLE
Add ability to configure metricsPrefix per source

### PR DESCRIPTION
**Pull request by loburm**

### CODE REVIEW

File: prometheus-to-sd/main.go [hunk 1]
```diff
@@ -69,19 +69,19 @@ func main() {
 	flag.Set("logtostderr", "true")
 	flag.Var(&source, "source", "source(s) to watch in [component-name]:http://host:port/path?whitelisted=a,b,c&podIdLabel=d&namespaceIdLabel=e&containerNameLabel=f format")
 	flag.Var(&dynamicSources, "dynamic-source",
-		`dynamic source(s) to watch in format: "[component-name]:http://:port/path?whitelisted=metric1,metric2&podIdLabel=label1&namespaceIdLabel=label2&containerNameLabel=label3". Dynamic sources are components (on the same node) discovered dynamically using the kubernetes api.`,
+		`dynamic source(s) to watch in format: "[component-name]:http://:port/path?whitelisted=metric1,metric2&podIdLabel=label1&namespaceIdLabel=label2&containerNameLabel=label3&metricsPrefix=prefix". Dynamic sources are components (on the same node) discovered dynamically using the kubernetes api.`,
```
Comment (x13n) [inline prometheus-to-sd/main.go]: Please explain metricsPrefix semantics in the help text.

Comment (loburm) [reply prometheus-to-sd/main.go]: Updated documentation with information about metricsPrefix.

**Comment by loburm** [comment]
Bumped version of prometheus-to-sd to 0.5.0 in Makefile.

**Comment by x13n** [comment]
LGTM, although Travis complains about formatting.

**Pull request merged**